flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2Branch.java

```diff
@@ -46,12 +46,19 @@ public class TestFlinkIcebergSinkV2Branch extends TestFlinkIcebergSinkV2Base {
   static final HadoopCatalogExtension CATALOG_EXTENSION =
       new HadoopCatalogExtension(DATABASE, TestFixtures.TABLE);
 
-  @Parameter(index = 0)
-  private String branch;
+  @Parameter(index = 4)
+  protected String branch;
 
-  @Parameters(name = "branch = {0}")
+  @Parameters(name = "branch = {4}")
   public static Object[][] parameters() {
-    return new Object[][] {new Object[] {"main"}, new Object[] {"testBranch"}};
+    return new Object[][] {
+      new Object[] {
+        FileFormat.AVRO, 1, false, TableProperties.WRITE_DISTRIBUTION_MODE_NONE, "main"
+      },
+      new Object[] {
+        FileFormat.AVRO, 1, false, TableProperties.WRITE_DISTRIBUTION_MODE_NONE, "testBranch"
+      }
+    };
```

Comment:
`format`, `parallelism`, `partitioned`, `writeDistributionMode`  from `TestFlinkIcebergSinkV2Base` are added to avoid type-mismatch between the inherited parameter `format` (FileFormat) and `branch` (String) because they are inherited from the parent (`TestFlinkIcebergSinkV2Base`) class.